### ISSUE TITLE
Summary of Calls

### ISSUE THREAD

**Issue by avinashbarnwal** [avinashbarnwal/aftXgboostPaper#1]
09/11/2019

- Resolving Survreg Error- 
![Survreg-Error](https://user-images.githubusercontent.com/6061417/64748716-1796ce00-d4e1-11e9-95c1-06ec268acbd7.png). I will be asking the author about the error.
- How to do cross-validation for 33 datasets as we have fold-wise datasets. Generally, for comparing all models, we require extra test set to check the performance of all models. But we are approaching like this - 

Below Picture is best
Fold1 Fold2 Fold3 Fold4 Fold5
One Fold -  Fold1 is considered as **Test dataset** and Rest from Fold 2 -5 are considered training the dataset and all hyperparameter optimization is done on that.
Similarly, Fold 2 becomes new test dataset and rest of datasets become the training dataset and all hyperparameter optimization is done on that. 

Based on the above observation we are going to have different hyperparameters for each fold.

- Write a similar function for model selection - https://github.com/tdhock/species-variable-selection/blob/master/cv.R

**Comment by tdhock** [comment]
you used survreg with dist="lognormal" which means your outputs should be all positive.. is that true? The outputs.csv.gz files have upper/lower limits which are sometimes negative, so for these data you should use "normal" instead.

**Comment by chyunsu3** [comment]
@tdhock Are these limits in log scale? My understanding is that survival times are positive (time-to-event).

**Comment by tdhock** [comment]
all the outputs.csv.gz files are limits on log scale

**Comment by avinashbarnwal** [comment]
This problem is nearly solved. I had to change the logic for finding the infinite values and had to remove 0 variance variables.

**Comment by avinashbarnwal** [comment]
09/27/2019

- @hcho3 gave me AWS EC2 access.
- Semi-Parametric AFT discussion.

**Comment by avinashbarnwal** [comment]
10/3/2019

- Create a folder for results
- Cross-Validation for Xgboost
- Implement Mboost corrected codes
- Share MRE with prof. Toby

**Comment by tdhock** [comment]
- MRE of mboost error + session info, please post as new issue
- please rearrange folder structure for result files -- would be nice if they were consistent with neuroblastoma-data/data
- please compute IntervalRegressionCV predictions and compare with files in neuroblastoma-data repo
